webapp/src/main/webapp/iaf/frankdoc/js/controllers.js

```diff
@@ -20,11 +20,24 @@ angular.module('iaf.frankdoc').controller("main", ['$scope', '$http', 'propertie
 			}
 			for(i in elements) {
 				var element = elements[i];
+				addAttributeActive(element);
 				$scope.elements[element.fullName] = element;
 			}
 		}
 	});
 
+	function addAttributeActive(element) {
+		attributeActive = {
+			name: "active",
+			description: "If defined and false, then this element and all its children are ignored"
```

Comment:
If defined and empty or false

Reply:
Empty wordt afgekeurd door de XSD. Als je doet active="" dan komt daar een rode kringel omheen. Als je doet active= dan ook.

Reply:
Ik denk daarom dat de tekst goed is zoals ik hem hierboven geschreven heb.

Reply:
Niels, ik heb de tekst aangepast zoals je hebt voorgesteld.